apps/studio/src/features/dashboard/components/FolderSettingsModal/FolderSettingsModal.tsx

```diff
@@ -1,6 +1,7 @@
 import { Suspense } from "react"
```

Comment:
More on the UI - should we keep the existing look but add the parent path prefix as part of the gray box?

![Screenshot 2024-12-19 at 11.30.42 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/AWVfQY5lNPJPEvYNU3KB/64db9f0d-621c-4863-ba26-228662ae3771.png)

Reply:
had a brief discussion with Chin and it seems like it will be too long, especially when considering the parent permalink

Reply:
actually the figma design is to have the infobox and not this, refer to the figma [here](https://www.figma.com/design/g7LnLn4IUcUC9dYxgsHEJM/Isomer-Studio-working-file?node-id=7037-20229&m=dev)
<img width="730" alt="image" src="https://github.com/user-attachments/assets/8617e0b3-52f7-4aa7-8e14-6fdb0b0e74c5" />

Reply:
Yea that's fine. What I felt helped was having the <example.com>/ or the site domain in front

For now a simple fix would be to keep the `your-site.gov.sg/` in front of the permalink

Reply:
we shuold change everything at once but idt that should be a blocker here